packages/core/src/components/Accordion/Item.tsx

```diff
@@ -14,7 +14,7 @@ export type Props = {
   /** Index amongst a collection of accordion items. */
   index?: number;
   /** Title of the accordion item. */
-  title?: string;
+  title?: React.ReactNode;
```

Comment:
maybe update the type to be `string | React.ReactNode`

Reply:
i originally had that, but i noticed Storybook only showed `ReactNode` as the type. then i thought that `string` is actually encapsulated under `ReactNode`? https://basarat.gitbooks.io/typescript/docs/jsx/react.html seems to verify (if you search ReactNode)

Reply:
dope! forgot about that 👍